diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index 289a924..4f03936 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -796,7 +796,7 @@
 	},
 
 	getEditingArea: function getEditingArea() {
-		return _reactDom2.default.findDOMNode(this.editingArea);
+		return this.editingArea;
 	},
 
 	getEditor: function getEditor() {
